controller/volume_controller.go

```diff
@@ -1573,19 +1573,17 @@ func (vc *VolumeController) ReconcileVolumeState(v *longhorn.Volume, es map[stri
 			if err != nil {
 				return err
 			}
-			if !canIMLaunchReplica {
+			if canIMLaunchReplica {
```

Comment:
In `canInstanceManagerLaunchReplica`, `ds.IsNodeDownOrDeleted` will check if node is down/gone or exists but not ready. 

Is it still valuable to rely on `node down/gone` only to determine the replica should be stopped instead of just removing the code here. This is still helpful for improving resilience if the node is really gone instead of timeout only. WDYT?

Reply:
It seems already discussed at https://github.com/longhorn/longhorn-manager/pull/1691#discussion_r1099626358, then it's OK to keep the current implementation.